Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.cpp

```diff
@@ -546,22 +546,6 @@ void VideoFrame::copyTo(std::span<uint8_t> destination, VideoPixelFormat pixelFo
     callback({ });
 }
 
-void VideoFrame::paintInContext(GraphicsContext& context, const FloatRect& destination, const ImageOrientation& destinationImageOrientation, bool shouldDiscardAlpha)
```

Comment:
Please don't remove this. Still used by the GStreamer player afaik.

Reply:
Yeah, sorry, that was from another work item.. restored.